taf/log.py

```diff
@@ -9,23 +10,34 @@
 logger = logging.getLogger('taf')
 logger.setLevel(taf.settings.LOG_LEVEL)
 
+
+
 if taf.settings.ENABLE_CONSOLE_LOGGING:
   console_handler = logging.StreamHandler()
   console_handler.setLevel(taf.settings.CONSOLE_LOGGING_LEVEL)
   console_handler.setFormatter(formatter)
   logger.addHandler(console_handler)
 
+
 if taf.settings.ENABLE_FILE_LOGGING:
-  file_handler = logging.FileHandler(taf.settings.LOG_FILENAME)
+  logs_location = taf.settings.LOGS_LOCATION
+  if logs_location is None:
+    logs_location = Path('/var/log')
+    if not logs_location.is_dir():
+      logs_location = Path.home() / '.taf'
+      logs_location.mkdir(exist_ok=True)
+  else:
+    logs_location = Path(logs_location)
+  file_handler = logging.FileHandler(str(logs_location / taf.settings.LOG_FILENAME))
   file_handler.setLevel(taf.settings.FILE_LOGGING_LEVEL)
   file_handler.setFormatter(formatter)
   logger.addHandler(file_handler)
 
-if taf.settings.SEPARATE_ERRORS:
-  error_handler = logging.FileHandler(taf.settings.ERROR_LOF_FILENAME)
-  error_handler.setLevel(taf.settings.ERROR_LOGGING_LEVEL)
-  error_handler.setFormatter(formatter)
-  logger.addHandler(error_handler)
+  if taf.settings.SEPARATE_ERRORS:
```

Comment:
Should lines 36-40. be indented?

Reply:
I moved this `if  taf.settings.SEPARATE_ERRORS` inside `if taf.settings.ENABLE_FILE_LOGGING` (doesn't really make sense to create `.err` if we should not log to files.

Reply:
I see now, never-mind.